demos/common/export_models/README.md

```diff
@@ -0,0 +1,102 @@
+# Exporting GEN AI models for deployments in OpenVINO Model Server
+
+This script automates exporting models from Hugging Faces hub or fine-tunned in Pytorch format to the models repository for deployment for the model serving.
+In one step it prepares a complete set of resources in the models repository for a supported GenAI use case.
+
+```bash
+pip install -q -r demos/common/export_models/requirements.txt
```

Comment:
I see that this readme is not exposed to sphinx, therefore I assume this readme file will be read on github. If so, I think the reader try to run these commands from within the context of demos/common/export_models directory.